### DISCUSSION TITLE
Set Sendgrid CNAMEs to Vercel DNS settings

### DISCUSSION THREAD

**Discussion by shami-dev** [vercel/community#2794] [Help] [answered]
### Summary

I have a website that is hosted in Vercel. To send form data to my email address, I use SendGrid. It provides three CNAMEs that I should set in Vercel. So, there are three fields: TYPE, HOST, and VALUE. When I paste them, it shows errors. How to do it in a proper way?

### Steps to Reproduce

<img width="1235" alt="vercel-dns-sett" src="https://github.com/vercel/community/assets/114109736/93464a8e-af7d-439e-bcdc-2826193b715a">

<img width="1137" alt="sendgrid-cnames" src="https://github.com/vercel/community/assets/114109736/d8dde4db-1c31-491a-817e-8b1e88cec8a2">

**Comment by swarnava** [comment, marked as answer]
You likely need to enter `Host` details in `Subdomain` & `value` details in `Value`

**Reply by shami-dev** [reply]
Thanks @swarnava for the response. It accepted only one record. I couldn't add all three.

**Reply by swarnava** [reply]
What is the error you are getting? You can reveal the host and value so we can see them as they aren't private / sensitive data.

**Reply by shami-dev** [reply]
As I understand, the mismatch error between these pairs. 
Oh, didn't know that it's not sensitive.
<img width="1093" alt="sendgrid-cnames2" src="https://github.com/vercel/community/assets/114109736/f02dc515-3bfe-4ab8-87c3-0415b9f9308f">

**Reply by swarnava** [reply]
Can you make sure you are only entering `em5376` , `s1._domainkey` and `s2._domainkey` in `subdomain` ? You will not need to enter `europe.com`

**Reply by shami-dev** [reply]
Let me check it again.

**Reply by shami-dev** [reply]
Also, I have a CNAME record by default: cname.vercel-dns.com
<img width="1251" alt="vercel-dns2" src="https://github.com/vercel/community/assets/114109736/91988112-e9a8-43b0-baf1-0ced1de602df">

This is after the verification. Probably, I should wait..
<img width="1176" alt="sengrid-dns3" src="https://github.com/vercel/community/assets/114109736/b8a985f1-26e3-4dd3-9c01-00360bccfa11">

**Reply by swarnava** [reply]
Do you own europe.com domain and currently the Nameserver is being pointed to Vercel NameServer? Otherwise we won't be able to setup the DNS.

**Reply by shami-dev** [reply]
No, I don't. I'm not an owner...
It seems that I should pay for using the domain name's email service.

**Reply by swarnava** [reply]
You can only setup email server to the domain that you own. You can purchase a domain and setup from scratch. :)

**Reply by shami-dev** [reply]
I purchased the domain but turned off a subscription for the email service. Probably, I need to reactivate it. 
Thank you for your help! 👍
